include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -44,76 +47,92 @@
 namespace seqan3
 {
 
```

Comment:
Can you prefix this block with 
```
// ------------------------------------------------------------------
// member exposure
// ------------------------------------------------------------------
```

And document the entities like in `alphabet/detail/member_exposure.hpp` (enclose them in a group named "Helpers ..." and use the wording from there, too?